website/content/docs/agent/config/index.mdx

```diff
@@ -51,9 +51,12 @@ required ports and their default settings.
 
 ## Reloadable Configuration
 
-Reloading configuration does not reload all configuration items. The
-items which are reloaded include:
+The following agent configuration options are reloadable at runtime.
+To make a Consul agent automatically reload these options when changed in its configuration files on disk,
+start the agent with its [`auto_reload_config` configuration option](/consul/docs/agent/config/config-files#auto_reload_config) set to `true`.
+You can also manually trigger reload with the [reload command](/commands/reload).
```

Comment:
```suggestion
You can run the [`consul reload` command](/commands/reload) to manually reload configurations in the configuration directory.

To configure the agent to automatically reload configurations when files are updated on disk, set the [`auto_reload_config` configuration option](/consul/docs/agent/config/config-files#auto_reload_config) parameter to `true`. 

The following agent configuration options are reloadable at runtime: 
```
We should put "the following . . ." statements as close to the list of items as possible. I also took a stab at trimming the long sentence about the auto reload configuration.

Reply:
I think it's important to lead by mentioning that only some options are reloadable to set expectations upfront.

Perhaps I can do something like this?
```suggestion
Some agent configuration options are reloadable at runtime.
You can run the [`consul reload` command](/commands/reload) to manually reload supported options from configuration files in the configuration directory.
To configure the agent to automatically reload configuration files updated on disk,
set the [`auto_reload_config` configuration option](/consul/docs/agent/config/config-files#auto_reload_config) parameter to `true`.

The following agent configuration options are reloadable at runtime: 
```